src/hotspot/share/ci/ciInstanceKlass.cpp

```diff
@@ -729,6 +730,26 @@ void ciInstanceKlass::dump_replay_instanceKlass(outputStream* out, InstanceKlass
   }
 }
 
+GrowableArray<ciInstanceKlass*>* ciInstanceKlass::transitive_interfaces() {
+  if (_transitive_interfaces == NULL) {
+    GUARDED_VM_ENTRY(
+            InstanceKlass* ik = get_instanceKlass();
```

Comment:
A candidate for `compute_transitive_interfaces()` helper method?

Reply:
Done in new commit.